mesatee_services/tdfs/internal/client/src/tdfs_client.rs

```diff
@@ -140,8 +142,16 @@ impl TDFSClient {
         let key_resp = client.request_get_key(&file_info.key_id)?;
         let key_config = key_resp.config;
         let access_path = &file_info.access_path;
-        let ciphertxt = fs::read(access_path)
-            .map_err(|_| mesatee_core::Error::from(mesatee_core::ErrorKind::IoError))?;
+        let mut f = fs::File::open(access_path)?;
+        let mut ciphertxt: Vec<u8> = Vec::new();
```

Comment:
Use with_capacity to initialize the vec.

Reply:
fixed.